app/controllers/flood_risk_engine/enrollments/exemptions_controller.rb

```diff
@@ -0,0 +1,32 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  class Enrollments::ExemptionsController < ApplicationController
+
+    def show
+      destroy # allows exemptions to be removed when JavaScript is disabled.
+    end
+
+    def destroy
+      enrollment.exemptions.destroy(exemption)
+      step_back if enrollment.exemptions.empty?
+      redirect_to enrollment_step_path(enrollment, enrollment.current_step)
+    end
+
+    private
+
+    def enrollment
```

Comment:
yeah its a good technique  ... and sure the cost is not worth worrying about .. was just curious